e2e-tests/cypress/support/insurance/check-your-business-summary-list.js

```diff
@@ -223,6 +233,50 @@ const checkYourBusinessSummaryList = ({
     const { expectedKey, expectedChangeLinkText } = getSummaryListField(fieldId, FIELDS.TURNOVER);
     const expectedValue = `${application.EXPORTER_BUSINESS[fieldId]}%`;
 
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
+  },
+  [USING_BROKER]: () => {
+    const fieldId = USING_BROKER;
+
+    const { expectedKey, expectedChangeLinkText } = getSummaryListField(fieldId, FIELDS.BROKER);
+    const expectedValue = application.EXPORTER_BROKER[fieldId];
+
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
+  },
+  [NAME]: () => {
+    const fieldId = NAME;
+
+    const { expectedKey, expectedChangeLinkText } = getSummaryListField(fieldId, FIELDS.BROKER);
+    const expectedValue = application.EXPORTER_BROKER[fieldId];
+
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
+  },
+  [ADDRESS_LINE_1]: () => {
+    const fieldId = ADDRESS_LINE_1;
+
+    const expectedKey = FIELDS.BROKER[fieldId].SUMMARY.TITLE;
+
+    const { summaryList } = checkYourAnswersPage;
+    const row = summaryList[fieldId];
+
+    cy.checkText(
+      row.key(),
+      expectedKey,
+    );
+
+    // as html, cannot use checkText so checking contains following fields
```

Comment:
We've got 2 tests for this, is it needed in both tests?

Reply:
where is the second one?

Reply:
Ignore me, seems I confused myself 🙃